fiftyone/core/stages.py

```
@@ -8169,10 +8174,11 @@ def load_view(self, sample_collection, reload=False):
 
         try:
             last_dataset = fod.load_dataset(name)
+            last_dataset.reload()
         except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)